evm_arithmetization/src/cpu/kernel/asm/transactions/common_decoding.asm

```diff
@@ -226,7 +226,12 @@ insert_accessed_storage_keys_with_original_value:
 after_read:
     %stack (value, addr, key, retdest) -> ( addr, key, value, retdest)
     %insert_accessed_storage_keys
-    %pop2
+    // stack: cold_access, value_ptr, value, retdest
```

Comment:
I think the whole function `insert_accessed_storage_keys_with_original_value` is obsolete now, since it's handled directly in `insert_accessed_storage_keys`.

Reply:
Hm what do you mean? We still need to insert the storage keys in the access list.

Reply:
Ah you're right, nevermind.